build/rpms/cri-tools.spec

```diff
@@ -0,0 +1,21 @@
+Name: cri-tools
+Version: OVERRIDE_THIS
```

Comment:
Forgot to change here?

Reply:
no, they all say that 

https://github.com/kubernetes/kubernetes/blob/master/build/rpms/kubernetes-cni.spec#L2
https://github.com/kubernetes/kubernetes/blob/master/build/rpms/kubectl.spec#L2